sources/model/GGRealTimeMontior.m

```diff
@@ -323,8 +323,8 @@ - (void)webSocketConnectWithCompletionHandler:(void (^)(BOOL success, NSError *e
     
     NSDictionary *connectionOptions = @{@"log": @(self.logsEnabled), @"forceWebsockets":@YES, @"secure": @(self.useSSL), @"reconnects":@NO, @"cookies":@[], @"connectParams":connectionParams};
 
-    SocketManager* manager = [[SocketManager alloc] initWithSocketURL:[NSURL URLWithString:server] config:connectionOptions];
```

Comment:
this guy is now retained and holds the socket on.

Reply:
https://stackoverflow.com/questions/48551895/tried-connecting-on-an-already-connected-socket-socket-io-client-swift